client-app/pages/account/profile.vue

```diff
@@ -117,20 +143,15 @@ const { defineField, errors, isSubmitting, meta, handleSubmit, resetForm } = use
 
 const [firstName] = defineField("firstName");
 const [lastName] = defineField("lastName");
-const [email] = defineField("email");
+const [defaultLanguage] = defineField("defaultLanguage");
+const [currencyCode] = defineField("currencyCode");
 
 const onSubmit = handleSubmit(async (data) => {
-  responseErrors.value = [];
```

Comment:
Is that really no needed anymore? If so, seems that `responseErrors` is not being used anywhere else, better to delete.

Reply:
Yes, because `updateContact` mutation returns updated contact not `IdentityError`.